package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-r9+UiPsezvyDSPb7IvRk4ypYs2s=",
       "dev": true,
       "requires": {
-        "acorn": "^3.0.4"
+        "acorn": "3.3.0"
```

Comment:
package-lock.json looks good to me since package.json wasnt change, tests pass, and this package is used as a static file